tests/resources.yaml

```diff
@@ -16,7 +16,7 @@ profile_model:
   - text.json
   - paper.pdf
   - code.py
-  - amr.json
+  - ground_truth_model_card.json
```

Comment:
I think this should be optional

Reply:
the only way we can make it optional atm is remove it from this yaml and do the optional check you suggested below (but that kind of defeats the purpose of resources.yaml right?)

Reply:
Maybe the `resources.yaml` file should have an `optional` section for various tests. Let's discuss a bit during standup

Reply:
done